### PR TITLE
Bug 1111981 - [v2.2] Investigate intermittent failure in test_sms_with_p...

### PR DESCRIPTION
**Pull request by chirarobert**
...icture_attached.py and test_add_photo_to_contact.py

### CODE REVIEW

**Comment by try-server-hook** [comment]
[chirarobert chirarobert](https://github.com/chirarobert) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=6fab543316fb)

File: tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/crop_view.py [hunk 1]
```diff
@@ -19,14 +19,16 @@
...
+    _edit_preview_canvas_locator = (By.ID, 'edit-preview-canvas')
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
         Wait(self.marionette).until(expected.element_displayed(
             Wait(self.marionette).until(expected.element_present(
-                *self._crop_view_locator))))
+                *self._edit_preview_canvas_locator))))
+        import time
+        time.sleep(2)
```
Comment (bebef1987) [inline tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/crop_view.py]: Why the sleep here?

Can you add a comment to explain why we need a hardcoded sleep

**Comment by try-server-hook** [comment]
[chirarobert chirarobert](https://github.com/chirarobert) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=723eea10ae6f)

**Comment by viorelaioia** [comment]
Let's see an adhoc run: http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk.ui.adhoc/623/console

File: tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/crop_view.py [hunk 2]
```diff
@@ -19,14 +19,18 @@
...
 
     def __init__(self, marionette):
         Base.__init__(self, marionette)
         Wait(self.marionette).until(expected.element_displayed(
             Wait(self.marionette).until(expected.element_present(
-                *self._crop_view_locator))))
+                *self._edit_preview_canvas_locator))))
+        # I have tried waiting for all the elements in the UI, but the crash
+        # still occurs so the only solution I found was the hardcoded sleep - Bug 1111981
+        import time
```
Comment (viorelaioia) [inline tests/python/gaia-ui-tests/gaiatest/apps/gallery/regions/crop_view.py]: pleae move this import abouve, in the imports section.

**Comment by try-server-hook** [comment]
[chirarobert chirarobert](https://github.com/chirarobert) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=a6202b9f3321)

**Comment by chirarobert** [comment]
The adhoc was failing because the branch needed a rebase.

http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk.ui.adhoc/624/console

**Comment by try-server-hook** [comment]
[chirarobert chirarobert](https://github.com/chirarobert) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=8d3e0da6932b)

**Comment by bebef1987** [comment]
http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk.ui.adhoc/625/

**Comment by chirarobert** [comment]
http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk.ui.adhoc/630/console

Started a new adhoc because the on @bebef1987 posted crashed.

**Comment by bebef1987** [comment]
http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk.ui.adhoc/645/console

**Comment by viorelaioia** [comment]
http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk.ui.adhoc/647/console

**Comment by viorelaioia** [comment]
http://jenkins1.qa.scl3.mozilla.com/view/UI/job/flame-kk.ui.adhoc/652/console

**Pull request merged**